README.md

```diff
@@ -62,3 +62,13 @@ ngrok http 8443 -host-header="localhost:8443"
 
 Details about how to deploy the frontend code can be found on the [CC Wiki](https://wikijs.creativecommons.org/tech/cc-search/frontend) (Accessible to CC Staff only).
 
+## Contributing
+* Create a new branch with a meaningful name `git checkout -b branch_name`.<br />
```

Comment:
this kind of information is already widely available on the internet, including on Github own docs. We don't this it's necessary to include it in the Readme

Reply:
okay @brenoferreira